docs/_reporting/hubs/configure-scopes.md

```diff
@@ -17,9 +17,9 @@ Connect FinOps hubs to your billing accounts and subscriptions by configuring Co
 <details open markdown="1">
    <summary class="fs-2 text-uppercase">On this page</summary>
 
+- [🛠️ Configure exports manually](#️-configure-exports-manually)
```

Comment:
Why are you making this one first? Don't we want to promote managed exports more?

Reply:
Just because manual exports are the simplest to setup today and I believe most of the customers are going to still be prefer this option. At least until we have figured out the custom UI to deploy hubs. 
In my opinion, manual exports +  PowerShell command to backfill data is good for most of the scenarios.